pages/about.html

```diff
@@ -4,9 +4,48 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel = "stylesheet" href = "styles/styles.css">
   <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Shadows+Into+Light&display=swap" rel="stylesheet">
 </head>
-<body>
-  
+<body class="color">
+      <ul>
+        <li>
+            <a class= "decoration" href = "portfolio.html">PORTFOLIO</a>
```

Comment:
Based on W3 styling guide, it's better practice to not leave space before and after the `=`.
```suggestion
            <a class="decoration" href="portfolio.html">PORTFOLIO</a>
```